book/python/using-connectors.md

```diff
@@ -0,0 +1,193 @@
+
+If we run one of these scripts we'll see there are some required arguments listed. The first one should be familiar. `--application-module` is the same as the one seen when running a Wallaroo worker (if you need a review of those command line parameters check the [Running Wallaroo](/book/running-wallaroo/running-wallaroo.md) section). Providing this argument allows the script to autodetect most of your settings by reading the same application description that your wallaroo app runs with.
+
+The application module is not quite enough for the script to know what settings to use when connecting to wallaroo. Since your application may have more than one connector, the `--connector` argument is required for disambiguating which connector in your application setup this script is running as. Once you pass this in, we can autodetect the most common settings.
+
+Many connector scripts will have their own additional arguments, some may be required and others may be optional. In this case we have two, the host and port. These argument names prefix the connector name to ensure that conflicting argument names do not cause problems (should be able to pass all your wallaroo arguments in without conflict for example).
+
+To illustrate this, let's look at what the `udp_source` command line parameters might look like:
+
+```bash
+./wallaroo/connectors/udp_source --application-module celsius --connector celsius_feed --celsius_feed-host 127.0.0.1 --celsius_feed-port 6789
+```
+
+In this example, we are running a UDP source for our application which listens to a UDP socket on 127.0.0.1 port 6789 and forwards the data in each packet to wallaroo as a message.
+
+Since the included connector scripts are self-contained, feel free to copy it over to your application tree to use them directly and make path management a little easier.
+
+## Starting With Prebuilt Connectors
+
+We have a number of [prebuild connectors](https://github.com/WallarooLabs/wallaroo/tree/{{book.wallaroo_version}}/connectors/). Here is a short description of each one. These are all written so they can be copied and customized in your application source tree. You're still welcome to run them in place as well.
+
+### Kafka
+
+This connector provides Kafka source and sink support. This uses the [kafka library](https://pypi.org/project/kafka/). Consumer group based sources are supported to allow easier management of your source's progress through a topic.
+
+While Wallaroo already has built-in Kafka support, using the connector allows you to support the use of consumer groups and reuse any logic you might already have around offset management and consumer lifecycles. We're looking for feedback, so if your Kafka usecase doesn't seem to fit either option, please let us know.
```

Comment:
Can we add link to a way to contact us (could be a mailto:) for the "please let us know".